Packs/VirusTotal/Integrations/integration-VirusTotal_5.5.yml

```diff
@@ -520,34 +520,27 @@ script:
     }
     function getURLs(url) {
         var urls = [];
+        var nonUrls = []
         var urlList = argToList(url);
         for (z = urlList.length - 1; z >= 0; z--) {
             var isURL = urlList[z].match(/(?:(?:https?|ftp|hxxps?):\/\/|www\[?\.\]?|ftp\[?\.\]?)(?:[-\w\d]+\[?\.\]?)+[-\w\d]+(?::\d+)?(?:(?:\/|\?)[-\w\d+&@#\/%=~_$?!\-:,.\(\);]*[\w\d+&@#\/%=~_$\(\);])?/);
             if (isURL == null){
-                var i = z;
-                var url;
-                var optionalURL;
-                while(i>=0 && z > 0 && optionalURL == null) {
-                    i--;
-                    optionalURL = urlList.slice(i, z).join("").match(/(?:(?:https?|ftp|hxxps?):\/\/|www\[?\.\]?|ftp\[?\.\]?)(?:[-\w\d]+\[?\.\]?)+[-\w\d]+(?::\d+)?(?:(?:\/|\?)[-\w\d+&@#\/%=~_$?!\-:,.\(\);]*[\w\d+&@#\/%=~_$\(\);])?/);
-                }
-                if (optionalURL !== null) {
-                    urls.push(urlList.slice(i, z+1).join(","));
-                }
-                z = i;
-            } else {
+                nonUrls.push(urlList[z])
```

Comment:
why did you remove the URL manipulation code?

Reply:
This code was going over the rest of the list and trying to match the same regex from above, and caused bad URLs to be added to the list.
For example:
On this list - 
![image](https://user-images.githubusercontent.com/50294648/104885653-08dfe700-5971-11eb-9767-d53aa000b198.png)
The integration now returns an error- Invalid URL.
A deeper look will show it's trying to send the string 'https://www.google.com,demisto.com,google.com' as one URL to the service.
So it's a bug + I couldn't understand this logic. 
Why not just try to match each of the URLs in the list to the regex?